### ISSUE TITLE
Initialize error: Protocol error (Runtime.callFunctionOn): Target closed

### ISSUE THREAD

**Issue by FahadAltaf** [chrishubert/whatsapp-api#137]
![image](https://github.com/chrishubert/whatsapp-api/assets/5881092/b74edff6-7441-4015-8327-bf05d8d3a517)

**Comment by FahadAltaf** [comment]
Yes, deploy on Azure container instance. After the volume is mounted it starts giving this. I can start the session and can also see the files created in the volume mounted. But when I request the QR code it throughs this error

**Comment by DenysSamoiliuk** [comment]
we also get this error when you change the markup version of the WhatsApp page, only deleting and reactivating the session helps
